x-pack/plugins/alerting/server/lib/wrap_scoped_cluster_client.test.ts

```diff
@@ -134,6 +148,7 @@ describe('wrapScopedClusterClient', () => {
       scopedClusterClient,
       rule,
       logger,
+      abortController,
     });
     const wrappedSearchClient = wrappedSearchClientFactory.client();
```

Comment:
There was a test inside `x-pack/plugins/alerting/server/lib/create_abortable_es_client_factory.test.ts` (`throws error when search throws abort error`) that would be good to move to this test file.

Reply:
Good catch! Added back in this commit => https://github.com/elastic/kibana/pull/124374/commits/92bc827f922a7ba80486b163fe24af8e65247b58 with a few modifications (child client).